build.sbt

```
@@ -144,11 +144,13 @@ lazy val cache = crossProject("cache")(JSPlatform, JVMPlatform)
     libraryDependencies ++= Seq(
       Deps.svm % Provided,
       Deps.windowsAnsi
-    )
+    ),
+    classloadersForCustomProtocolTest(customProtocol.jvm),
   )
   .jsSettings(
     name := "fetch-js",
-    libs += Deps.cross.scalaJsDom.value
+    libs += Deps.cross.scalaJsDom.value,
+    classloadersForCustomProtocolTest(customProtocol.js),
```

This shouldn't be needed, the code you changed is JVM-only.
```suggestion
    libs += Deps.cross.scalaJsDom.value
```